Project/core/src/main/java/Controller/RecruiterActionController.java

```diff
@@ -0,0 +1,90 @@
+package Controller;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import Model.Board;
+import Model.BrainFact;
+import Model.Feature;
+import Model.Footstep;
+import Model.NormalCell;
+import Model.Token;
+
+public class RecruiterActionController {
+
+    /**
+     * Moves the recruiter
+     * 
+     */
+    public boolean move() {
+        return true;
+    }
+
+    public boolean mindSlip() {
+        // Check Valid Mindslip moves
+
+        // Send valid moves to view (recieve player move)
+
+        // Update board state
+
+        return true;
+    }
+
```

Comment:
Right now thinking about how to implement this. Check valid move currently adds mindslip possibilities whenever it does the check for a recruiter but we need to separate them still somehow. Still looks good to me!